docs/src/api-testing-java.md

```diff
@@ -0,0 +1,439 @@
+---
+id: api-testing
+title: "API testing"
+---
+
+Playwright can be used to get access to the [REST](https://en.wikipedia.org/wiki/Representational_state_transfer) API of
```

Comment:
REST -> HTTP?

Reply:
Rest is consistent with https://playwright.dev/docs/test-api-testing and we mention HTTP(s) below. Let's leave it for now.